structuredefinitions/Extension-UKCore-DiagnosticReportSupportingInfo.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-UKCore-DiagnosticReportSupportingInfo" />
+  <url value="http://hl7.org/fhir/5.0/StructureDefinition/extension-DiagnosticReport.supportingInfo" />
+  <version value="0.0.1" />
+  <name value="ExtensionUKCoreDiagnosticReportSupportingInfo" />
+  <title value="Extension UK Core DiagnosticReport SupportingInfo" />
+  <status value="draft" />
+  <date value="2023-11-28" />
+  <publisher value="HL7 UK" />
+  <contact>
+    <name value="HL7 UK" />
+    <telecom>
+      <system value="email" />
+      <value value="ukcore@hl7.org.uk" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="An extension to support adding additional info supporting info the diagnostic report." />
```

Comment:
"support adding additional info supporting info" doesn't make sense. Same in purpose.